config/environments/development.rb

```diff
@@ -10,7 +10,7 @@
   config.cache_classes = false
 
   # Do not eager load code on boot.
-  config.eager_load = false
+  config.eager_load = true
```

Comment:
This doesn't look like a framework default, maybe move this change to a separate commit where you explain why it's done?

Reply:
Having had a chat with Deb this is part of Zeitwerk's autoloading based on naming convention https://github.com/alphagov/signon/pull/1356/commits/46921427a34dca36e2fb4dc334b324ee4ac73424
With that in mind I've temporarily disabled this as zeitwerk is a lot of work to implement in publishing-api and will be brought in in a separate ticket, I'll make sure to add this as a 'first thing to do' in the ticket.